### PR TITLE
allow widgets to use customizer styles

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [TEC-3391]

This PR allows widgets to use customizer styles.

Depends on: https://github.com/moderntribe/tribe-common/pull/1507

**Artifact:** https://drive.google.com/file/d/1vlXTwarhF7PGnxjXw7pqr9Hok8ARck3Y/view?usp=sharing

[TEC-3391]: https://moderntribe.atlassian.net/browse/TEC-3391

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -354,6 +354,53 @@ public function get_css_template( $template ) {
 					color: <%= global_elements.accent_color %>;
 				}
 			';
+
+			// overrides for tec widgets/full/_events-list.pcss.
+			$template .= '
+				.tribe-events-widget .tribe-events-widget-events-list__event-row--featured .tribe-events-widget-events-list__event-date-tag-datetime:after {
+					background-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:361]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: e36960144e1dd221e8213538c6f61833</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -354,6 +354,53 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-events-widget .tribe-events-widget-events-list__view-more-link,
+				.tribe-events-widget .tribe-events-widget-events-list__view-more-link:visited {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:368]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: aaecc4d4290253edd2f74e04a6bde445</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -354,6 +354,53 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-events-widget .tribe-events-widget-events-list__view-more-link:active,
+				.tribe-events-widget .tribe-events-widget-events-list__view-more-link:focus,
+				.tribe-events-widget .tribe-events-widget-events-list__view-more-link:hover {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:376]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 01926ec495f9ee79ed3ab0ef23bdec6c</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 4]
```diff
@@ -354,6 +354,53 @@ public function get_css_template( $template ) {
...
+					border-bottom-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:377]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 3ff679632a5f434e7e79f35de2b76c24</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 5]
```diff
@@ -354,6 +354,53 @@ public function get_css_template( $template ) {
...
+				.tribe-theme-enfold .tribe-events-widget .tribe-events-widget-events-list__view-more-link:visited,
+				.tribe-theme-enfold .tribe-events-widget .tribe-events-widget-events-list__view-more-link:hover,
+				.tribe-theme-enfold .tribe-events-widget .tribe-events-widget-events-list__view-more-link:focus,
+				.tribe-theme-enfold .tribe-events-widget .tribe-events-widget-events-list__view-more-link:active,
+				.main_color .sidebar .tribe-events-widget .tribe-events-widget-events-list__view-more-link,
+				.main_color .sidebar .tribe-events-widget .tribe-events-widget-events-list__view-more-link:visited,
+				.main_color .sidebar .tribe-events-widget .tribe-events-widget-events-list__view-more-link:hover,
+				.main_color .sidebar .tribe-events-widget .tribe-events-widget-events-list__view-more-link:focus,
+				.main_color .sidebar .tribe-events-widget .tribe-events-widget-events-list__view-more-link:active {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:401]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: a10707bc353ee2fc02515fb45a4c5d2f</sub>

File: src/Tribe/Views/V2/Widgets/Widget_List.php [hunk 6]
```diff
@@ -80,6 +80,16 @@ class Widget_List extends Widget_Abstract {
 		],
 	];
 
+	/**
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Widgets/Widget_List.php:83]: :no_entry_sign: Doc comment for parameter "$arguments" missing

<sub>hash: c4c1045553e2dc32ecb1efbc265e194c</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)
:warning: 5 warning(s)

**Review by lucatume — DISMISSED** [review]

**Review by lucatume — APPROVED** [review]

**Pull request merged**